core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -702,6 +704,8 @@ public PropServerConfiguration(
 
             this.inputRoot = getString(properties, env, "cairo.sql.copy.root", null);
             this.backupRoot = getString(properties, env, "cairo.sql.backup.root", null);
+            this.snapshotRoot = getString(properties, env, "cairo.sql.snapshot.root", null);
+            this.snapshotDirTimestampFormat = getSnapshotTimestampFormat(properties, env);
```

Comment:
Do we want this to be configurable? If it is `yyyy-MM-dd` then will it mean that it will be only 1 snapshot per day? I'd rather have it hardcoded to a second precision